src/js/pages/JobsPage.js

```diff
@@ -0,0 +1,66 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {RouteHandler} from 'react-router';
+
+import Page from '../components/Page';
+import SidebarActions from '../events/SidebarActions';
+import TabsMixin from '../mixins/TabsMixin';
+
+class JobsPage extends mixin(TabsMixin) {
+  constructor() {
+    super();
```

Comment:
Because it's really easy to miss in the future. This component could evolve to having other mixins and it's really nice to not have to think about which methods the mixin uses from react's lifecycle if we just call super with arguments.